### DISCUSSION TITLE
Conform setup intent not working in release mode it is working fine in debug mode

### DISCUSSION THREAD

**Discussion by Zeeshan-progs** [flutter-stripe/flutter_stripe#1190] [Q&A]

**Comment by rsys-zeeshan-iqbal** [comment]
I am having same issue in my project 

No errors throw 
![carbon (1)](https://user-images.githubusercontent.com/101088061/227754740-56691483-6768-4505-bc54-d97a8e66ec08.png)


![carbon](https://user-images.githubusercontent.com/101088061/227754716-850e8208-5ce8-4657-9a10-6df57e032ef8.png)
